lxc/storage_volume.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/spf13/cobra"
 	"gopkg.in/yaml.v2"
 
-	"github.com/lxc/lxd/client"
+	lxd "github.com/lxc/lxd/client"
```

Comment:
This change can be dropped. It's something `goimports` does automatically.

Reply:
Oops, fixed now.